src/test/java/com/facebook/openwifirrm/optimizers/TestUtils.java

```diff
@@ -408,10 +417,37 @@ public static State createState(int channel, int channelWidth, String bssid) {
 		return createState(
 			new int[] {channel},
 			new int[] {channelWidth},
-			new String[] {bssid}
+			new int[] {DEFAULT_CHANNEL_WIDTH},
+			new String[] {bssid},
+			new int[][] {{}}
 		);
 	}
 
+	/**
+	 * Create a device state object with one radio and client RSSIs.
+	 *
+	 * @param channel channel number
+	 * @param channelWidth channel width in MHz
+	 * @param bssid bssid
```

Comment:
same here, if you are going to document any parameters with `@param` you should document all of them